### DISCUSSION TITLE
Twig - The cache tag

### DISCUSSION THREAD

**Discussion by WebVPF** [wintercms/winter#599] [Ideas]
Is there a cache twig tag in Winter 1.2?
Twig 3.2 added the `cache` tag https://twig.symfony.com/doc/3.x/tags/cache.html
Maybe add this to Winter?

**Comment by bennothommo** [comment]
Unfortunately we don't include the extra dependency that brings that tag in, and I don't think we have any plans at this stage.

**Reply by WebVPF** [reply]
@bennothommo can you tell me what theme you used when creating this screenshot?

![Screenshot Winter VSCode Extension](https://raw.githubusercontent.com/wintercms/vscode-extension/main/assets/screenshot.png)

**Reply by bennothommo** [reply]
@WebVPF sure thing - it's the [Behave Full Dark Theme](https://marketplace.visualstudio.com/items?itemName=Chrismettal.behave-full-theme) in VSCode :smiley:

**Comment by LukeTowers** [comment]
I'm not sure I see the point TBH, Twig templates are already cached, and if you want to add caching to your templates you're better off either caching the entire response or caching your data retrieval steps as that's typically more expensive than rendering a small fragment of your overall response.
